todo-app/src/components/TaskItem.js

```diff
@@ -0,0 +1,78 @@
+import React, { Component } from "react";
+import PropTypes from "prop-types"
+import { inject, observer } from "mobx-react"
+
+import Trash from "react-icons/lib/fa/trash"
+import Edit from "react-icons/lib/fa/edit"
+
+@inject("todosStore")
+@observer
+export default class TaskItem extends Component {
+
+    static propTypes = {
+        id: PropTypes.number,
+        description: PropTypes.string,
+        deadLine: PropTypes.string,
+        done: PropTypes.bool,
+        creatingDate: PropTypes.string
+    }
+
+    handleToggling = () => {
```

Comment:
Naming function should start with meaningful verbs for the real action that the function does. For example here it should be `toggleTodo`. For removing you can name it `removeTodo`. In general don't use **handle** because it is vague for what your function really does.